frontend/packages/console-dynamic-plugin-sdk/src/app/components/safety-first.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
  * Hook that ensures a safe asynchronnous setting of React state in case a given component could be unmounted.
  * (https://github.com/facebook/react/issues/14113)
  * @param initialState initial state value
- * @return An array with a pair of state value and it's set function.
+ * @returns An array with a pair of state value and it's set function.
```

Comment:
@TheRealJon AFAIK `@return` is a synonym for `@returns` right? Why are we changing these? For consistency?

Reply:
https://jsdoc.app/tags-returns.html

Reply:
I wonder if there is some way to configure blocking `@return` if we don't want this to happen in the future.

Reply:
You're right that `@return` is a valid tsdoc block, but the tsdoc parser wasn't properly recognizing it and was concatenating @return blocks with the previous block. It might be worth reporting a bug for this.